Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.12.0
+
+ARG CRYPTOMATOR_CLI_VERSION=0.4.0
+
+RUN adduser -D cryptomator && \
+    apk add --no-cache curl openjdk11-jre-headless && \
+    curl -sSL https://github.com/cryptomator/cli/releases/download/$CRYPTOMATOR_CLI_VERSION/cryptomator-cli-$CRYPTOMATOR_CLI_VERSION.jar > /usr/bin/cryptomator.jar && \
```

Comment:
Why are you not using wget which is included by default?

Reply:
Makes perfect sense, I am just more familiar with curl in-depth. [Already replaced curl with wget in my fork](https://github.com/pmjohann/cli/commit/14d10739f6a0cb04e457dd67558d83d982729f48), I'll start a new pull request with the changes as soon as I updated the readme as well related to your other comment.

Reply:
This PR gets automatically updates if you change it in the fork.